apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,9 +1,9 @@
 ack_generate_info:
-  build_date: "2022-12-08T18:57:06Z"
-  build_hash: 12246c7da82841b351ec7a9e1f139f9338f2784b
-  go_version: go1.17.5
-  version: v0.21.0
-api_directory_checksum: 9ccbf1f9bf7c5ee745f5b55fe465211dd2bb109a
+  build_date: "2023-02-02T22:44:37Z"
+  build_hash: b55ae8752ece381c383ffe5b388ed2147c6b30d8
+  go_version: go1.19.1
+  version: v0.23.1-dirty
```

Comment:
@RedbackThomson  I just realized that those PRs, were shipped with a dirty commit, is this normal coming from ack-bot?

Reply:
Oh for this one I pushed a commit on top with updated initialisms. Oops